adapter/config/default_config.go

```diff
@@ -146,6 +146,30 @@ var defaultConfig = &Config{
 			AllowPartialMessage: false,
 			PackAsBytes:         false,
 		},
+		Filters: filters{
+			Compression: compression{
+				Enabled: true,
+				Library: "gzip",
+				RequestDirection: requestDirection{
+					Enabled:              false,
+					MinimumContentLength: 30,
+					ContentType:          []string{"text/html", "application/json"},
```

Comment:
Can't we ship all the content types envoy allow bydefault?
https://www.envoyproxy.io/docs/envoy/v1.23.1/api-v3/extensions/filters/http/compressor/v3/compressor.proto#envoy-v3-api-msg-extensions-filters-http-compressor-v3-compressor

Reply:
+1

Reply:
@AmaliMatharaarachchi @VirajSalaka 
Can mention here also. But in the docs we are mentioning the supported content types under the supported values. 
Additional to that do we need to list all the supporting content types by default ?